### PR TITLE
adding Furnitures category page

### PR DESCRIPTION
**Pull request by kadarikalyani**
adding Furnitures category page

### CODE REVIEW

**Comment by abhinaykrupa** [comment]
@kadarikalyani i requested few changes to this commit ... go thru my comments and make changes accordingly ... this is your priority for today !

**Comment by kadarikalyani** [comment]
adding furniture category new page

**Comment by abhinaykrupa** [comment]
@kadarikalyani i dont think you commited the new changes and pushed them to remote origin/kalyani branch ... can you do that !

**Comment by kadarikalyani** [comment]
adding Furnitures category new page

File: application/controllers/Furnitures.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        .'affordable, apparel, area rugs, bargains, bedding, bedroom, cheap, deals, dining room, discounts, electronics, entertainment, european home furnishings, free, furniture, home décor, home furnishings, home furniture, home office, jewelry, liquidate, liquidator, living room , low prices, lowest, '
+        . 'prices, mattresses, merchandise, modern, modern decor, modern furnishings, modern furniture, name brand, online shopping, online store, outdoor furniture, overstock, pottery barn furniture, pottery barn home decor,'
+        .' quality, sale, savings, transitional, unique home furnishings, unique home furniture, values, watches, wholesale, youth,Ashley Furniture HomeStore,best price,contemporary,european home furniture,home decor';
+        $this->websiteDescriptionsTitle = 'Top Websites to Shop For Furniture Online';
+        $this->websiteDescriptionsIntro = 'Finding your dream home is a tedious task. we have listed americas few TOP Real Estate Portals below that might ease your home search.';
```
Comment (abhinaykrupa) [inline application/controllers/Furnitures.php]: @kadarikalyani this is still not changed !  please come up with something that makes sense here

**Comment by abhinaykrupa** [comment]
@kadarikalyani  .... also you forgot to add this furniture page to the sitemap.xml

**Comment by abhinaykrupa** [comment]
@kadarikalyani  i added a websiteDescriptionsIntro  to the furnitures controllers ... get the latest changes from master branch into your kalyani branch and you will see the new text.

**Pull request merged**